### PR TITLE
hate hypixel

### PR DESCRIPTION
**Pull request by SubAt0m1c**

### CODE REVIEW

File: src/main/kotlin/me/odinmain/features/impl/render/RenderOptimizer.kt [hunk 1]
```diff
@@ -139,13 +139,5 @@ object RenderOptimizer : Module(
...
-    private fun getHealerFairyTextureValue(armorStand: EntityArmorStand?): String? {
-        return armorStand?.heldItem
-            ?.tagCompound
-            ?.getCompoundTag("SkullOwner")
-            ?.getCompoundTag("Properties")
-            ?.getTagList("textures", 10)
-            ?.getCompoundTagAt(0)
-            ?.getString("Value")
-    }
+    private fun getHealerFairyTextureValue(armorStand: EntityArmorStand?): String? = armorStand?.heldItem?.skullTexture
```
Comment (odtheking) [inline src/main/kotlin/me/odinmain/features/impl/render/RenderOptimizer.kt:142]: Doesn't need to be a function

File: src/main/kotlin/me/odinmain/features/ModuleManager.kt [hunk 2]
```diff
@@ -89,7 +90,17 @@ object ModuleManager {
...
-        tickTasks.removeAll {
+        tickTasks.tickTaskTick()
+    }
+
+    @SubscribeEvent
+    fun onServerTick(event: RealServerTick) {
+        tickTasks.tickTaskTick(true)
+    }
+
+    private fun MutableList<TickTask>.tickTaskTick(server: Boolean = false) {
```
Comment (odtheking) [inline src/main/kotlin/me/odinmain/features/ModuleManager.kt:101]: can get a better naming?

File: src/main/kotlin/me/odinmain/features/impl/dungeon/BloodCamp.kt [hunk 3]
```diff
@@ -212,20 +214,20 @@ object BloodCamp : Module(
...
+        "ewogICJ0aW1lc3RhbXAiIDogMTcxOTYwNjIxMjc1NSwKICAicHJvZmlsZUlkIiA6ICI2NGRiNmMwNTliOTk0OTM2YTY0M2QwODEwODE0ZmJkMyIsCiAgInByb2ZpbGVOYW1lIiA6ICJUaGVTaWx2ZXJEcmVhbXMiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvOWZkNjFlODA1NWY2ZWU5N2FiNWI2MTk2YThkN2VjOTgwNzhhYzM3ZTAwMzc2MTU3YjZiNTIwZWFhYTJmOTNhZiIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9", //i hate hypixel
+        "ewogICJ0aW1lc3RhbXAiIDogMTcxOTYwNjIzOTU4NiwKICAicHJvZmlsZUlkIiA6ICJhYWZmMDUwYTExOTk0NzM1YjEyNDVlNDk0MGFlZjY4NCIsCiAgInByb2ZpbGVOYW1lIiA6ICJMYXN0SW1tb3J0YWwiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZTVjMWRjNDdhMDRjZTU3MDAxYThiNzI2ZjAxOGNkZWY0MGI3ZWE5ZDdiZDZkODM1Y2E0OTVhMGVmMTY5Zjg5MyIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9" //I really hate hypixel
```
Comment (odtheking) [inline src/main/kotlin/me/odinmain/features/impl/dungeon/BloodCamp.kt:226]: does that mean u got all of them?

Comment (SubAt0m1c) [reply src/main/kotlin/me/odinmain/features/impl/dungeon/BloodCamp.kt:226]: yeah

**Pull request merged**